robyn/utils.py

```diff
@@ -0,0 +1,5 @@
+def decode_bytearray(req):
+    """
+    Decodes a bytearray into a UTF-8 string
+    """
+    return bytearray(req["body"]).decode("utf-8")
```

Comment:
I would suggest keeping this in the `dependencies` file only as this is only used there for now.

Reply:
Also, smart implementation here! Love it!

Reply:
Thanks!